### DISCUSSION TITLE
Urgent! The css files on your server (use.fontawesome.com) are blocked by browser!

### DISCUSSION THREAD

**Discussion by tonydream888** [FortAwesome/Font-Awesome#20131] [General]
Our website enabled “Cross-Origin Isolation”, so all resources(css, font, etc) on your server (use.fontawesome.com) are blocked by browser(Firefox, etc)!

To fix the problem, please add the following http response header to your server:

Cross-Origin-Resource-Policy: cross-origin

that header will tell browser that your server give clear permission to other servers to access resources(css, etc) on your server.

Thanks!

**Comment by tagliala** [comment]
Converted to an issue here: #20132
